src/main/java/uk/ac/ebi/subs/agent/converters/BsdAttributeToUsiAttribute.java

```diff
@@ -20,10 +21,13 @@ public Attribute convert(uk.ac.ebi.biosamples.model.Attribute bsdAttribute) {
         usiAttribute.setValue(bsdAttribute.getValue());
         usiAttribute.setUnits(bsdAttribute.getUnit());
 
-        Term term = new Term();
         if(bsdAttribute.getIri() != null) {
-            term.setUrl(bsdAttribute.getIri());
-            usiAttribute.setTerms(Arrays.asList(term));
+            List<Term> terms = bsdAttribute.getIri().stream()
+                    .filter( bsdTerm -> bsdTerm != null)
```

Comment:
Can we call it bsdIri instead of bsdTerm, please?

Reply:
Yes